js/okx.js

```diff
@@ -4680,19 +4680,25 @@ module.exports = class okx extends Exchange {
         // WARNING: THIS WILL INCREASE LIQUIDATION PRICE FOR OPEN ISOLATED LONG POSITIONS
         // AND DECREASE LIQUIDATION PRICE FOR OPEN ISOLATED SHORT POSITIONS
         if ((leverage < 1) || (leverage > 125)) {
-            throw new BadRequest (this.id + ' setLeverage() leverage should be between 1 and 125');
+            throw new BadRequest (this.id + ' setLeverage () leverage should be between 1 and 125');
         }
         await this.loadMarkets ();
         const market = this.market (symbol);
-        const marginMode = this.safeStringLower (params, 'mgnMode');
-        params = this.omit (params, [ 'mgnMode' ]);
+        const marginMode = this.safeStringLower2 (params, 'marginMode');
```

Comment:
first, we're using `safeStringLower2` and passing a single key, second why not use `handleMarginModeAndParams` here?